components/molecules/m-product-call-to-action.vue

```diff
@@ -22,31 +22,20 @@
         />
       </div>
     </div>
-    <AAddToWishlist
-      class="text-action"
-      :product="product"
-    />
-    <AAddToCompare
-      class="text-action"
```

Comment:
`.text-action` style can be also removed.

Reply:
done